### PR TITLE
add-new-project

### PR DESCRIPTION
**Pull request by git-anurag-hub**

### CODE REVIEW

**Review by suubh — COMMENTED** [review]
Please add properly ,You just extended my code by adding the my code only . You only have to add -

> <li><i class="fas fa-code"></i> **_Description of your project_**.<a href="**_link of your project_**"> View here</a><li>

Clone again and create a new branch and then do the above mention changes only .

File: templates/index.html [hunk 1]
```diff
@@ -23,78 +29,129 @@
...
+              ><i class="fab fa-github fa-2x"></i
+            ></a>
+          </li>
+          <li>
+            <a href="https://www.linkedin.com/in/shubham-singh-356ba5168"
+              ><i class="fab fa-linkedin fa-2x"></i
+            ></a>
+          </li>
+          <li>
+            <i class="fas fa-code"> </i> This is the playground for the
```
Comment (git-anurag-hub) [inline templates/index.html:147]: See this line. This is the code I added.
By far the concern of extending your code. It is due to the use of the prettier and eslint which is widely used by developed.
```suggestion
            <i class="fas fa-code"> </i> This is the playground for the

```

**Review by git-anurag-hub — COMMENTED** [review]
..

**Comment by suubh** [comment]
Hello Anurag .I agree with your concern with prettier and most used by
developers . In future I will create an issue for robusting some sections
of  code . *This issue is for only adding your projects* .You have to add
only one line of code inside the class="project list".Just* <li><i
class="fas fa-code"></i> .......Description of your project......<a
href="....link .....">View here</a></li> .*
I totally agree with you ,hope you also understand the issue . Create a new
PR with only one line modification .
Thank You Anurag.
<div class="project-sec">
        <h3 class="project">Projects</h3>
        <div class="project-list">
          <ul>
            <li><i class="fas fa-code"> </i
> Horizontal and vertical navigation using HTML and CSS.<a href="
https://suubh.github.io/CSS-Learning/Tables/verticalnav.html">View here</a
></li>
            <li><i class="fas fa-code"> </i
> Dropdown navigation using HTML and CSS.<a href="
https://suubh.github.io/CSS-Learning/Tables/dropdown.html">View here</a></li
>
            <li><i class="fas fa-code"> </i> Some assignment work.<a href="
https://suubh.github.io/CSS-Learning/Assignment/index.html">View here</a></
li>
            <li><i class="fas fa-code"> </i
> Create amazing Github readme file for secret repository.Fork from here.<a
href="https://github.com/suubh/suubh">View here</a></li>
          </ul>
        </div>
      </div>

**Pull request closed without merging**